apps/editing-toolkit/editing-toolkit-plugin/error-reporting/src/handler.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Module variables
+ */
+
+/**
+ * Interval for error reports so we don't flood te endpoint. More frequent
+ * reports get throttled. If a request is made within this ms window, then
+ * it will be ignored and not sent, and the throttled counter will be incremented.
+ *
+ * @type {number} throttling interval (since the last request) in milliseconds.
+ */
+const REPORT_INTERVAL = 60000; //1 minute
+
+let throttled = 0;
+let lastReport = 0;
```

Comment:
ETK already pays the cost for lodash becasue it's used throughout. Might as well embrace that and use its `throttle` function here.

Reply:
Ahh, good point :)